shared/utils/signature.ts

```diff
@@ -0,0 +1,58 @@
+export const signatureStrokeSize = 8
+export const signatureStrokeThinning = 0.5
+export const signatureStrokeSmoothing = 0.5
+export const signatureStrokeStreamline = 0.5
+
+export const signatureOutputBoxWidthDefault = 500
+export const signatureOutputBoxHeightDefault = 500
+export const signatureOutputPaddingDefault = 10
+export const signatureOutputStrokeFillStyle = 'black'
+
+export const convertToSignatureStringOutput = (
+  input: [number, number, number][][],
+): string => {
+  return JSON.stringify(input)
+}
+
+export const convertToSignatureVectorArray = (
+  input: string,
+): [number, number, number][][] => {
+  return JSON.parse(input)
+}
+
+type BoundingBox = {
+  minX: number
+  minY: number
+  maxX: number
+  maxY: number
+}
+
+export const getBoundingBox = (
```

Comment:
suggestion: i wonder if the purpose of this function can be explained via a jsdocs comment as it seems like an implementation thing. This can be challenging to understand by future readers of the code why this is necessary and perhaps itll benefit readers to include some docs explaining its purpose.